ansible/templates/common/install.sh.j2

```diff
@@ -1,11 +1,17 @@
 {% if doc %}
-Install OMERO.web::
+Upgrade pip and install OMERO.web:
+
+.. parsed-literal::
+
+    /opt/omero/web//venv3/bin/pip install --upgrade pip
+    /opt/omero/web//venv3/bin/pip install "omero-web>=\ |version_web|"
+
 {% else %}
 #!/bin/bash
 
 set -e -u -x
 
+{{ virtualenv_path }}/bin/pip install --upgrade pip
+{{ virtualenv_path }}/bin/pip install "omero-web>=5.6.1"
```

Comment:
if this is an installation from scratch, is there a reason to specify a minimum version or can it be omitted as `pip install` should default to the latest version satisfying the package constraints?